File: Code/Python/LAB03_02.py
# Function to print even numbers 2 to 20 using for loop
def print_for():
    print("Using for loop: ", end="")
    for i in range(2, 21, 2):
        print(i, end=" ")
    print()

# Function to print even numbers 2 to 20 using while loop
def print_while():
    print("Using while loop: ", end="")
    i = 2
    while i <= 20:
        print(i, end=" ")
        i += 2
    print()

# Function to print even numbers 2 to 20 using do-while loop
def print_do_while():
    print("Using do-while loop: ", end="")
    i = 2
    while True:
        print(i, end=" ")
        i += 2
        if i > 20:
            break
    print()

def main():
    print_for()
    print_while()
    print_do_while()

if __name__ == "__main__":
    main()

File: Code/Cplus/LAB03_02/LAB03_02.cpp
#include <iostream>
using namespace std;

// Function to print even numbers 2 to 20 using for loop
void printFor() {
    cout << "Using for loop: ";
    for (int i = 2; i <= 20; i += 2) {
        cout << i << " ";
    }
    cout << endl;
}

// Function to print even numbers 2 to 20 using while loop
void printWhile() {
    cout << "Using while loop: ";
    int i = 2;
    while (i <= 20) {
        cout << i << " ";
        i += 2;
    }
    cout << endl;
}

// Function to print even numbers 2 to 20 using do-while loop
void printDoWhile() {
    cout << "Using do-while loop: ";
    int i = 2;
    do {
        cout << i << " ";
        i += 2;
    } while (i <= 20);
    cout << endl;
}

int main() {
    printFor();
    printWhile();
    printDoWhile();
    return 0;
}
